literature/literature.bib

```diff
@@ -19189,6 +20139,21 @@ @inproceedings{OZML:VaMoS11
 	tttags    = {SPL Testing},
 }
 
+@inproceedings{SRG:VaMoS11,
+	author    = {Schmid, Klaus and Rabiser, Rick and Gr{\"u}nbacher, Paul},
+	title     = {{A Comparison of Decision Modeling Approaches in Product Lines}},
+	booktitle = vamos,
+	pages     = {119--126},
+	location  = {Namur, Belgium},
+	url       = {https://dl.acm.org/doi/abs/10.1145/1944892.1944907},
```

Comment:
Omit this url because it is just a copy of the doi. In all such cases, we typically omit the URL because it does not contain any additional information.